nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ScanAttribute.java

```diff
@@ -75,21 +87,24 @@
             .defaultValue(MATCH_CRITERIA_ANY)
             .build();
     public static final PropertyDescriptor ATTRIBUTE_PATTERN = new PropertyDescriptor.Builder()
-            .name("Attribute Pattern")
```

Comment:
Must retain the previously existing name so as not to disturb existing configurations.  Andy's advice for the new property is good to follow going forward but it is wise to avoid changing old names.